.github/workflows/docker.yml

```diff
@@ -0,0 +1,15 @@
+name: Docker Image CI
```

Comment:
CookRight needs a multiarch build and requires an emulator, so this won't work for that. Not sure it makes sense to put this docker workflow in a template since build instructions vary.

Reply:
I can rename the repo, but I meant this as any microservice template (React, classifier, etc)